requirements.txt

```diff
@@ -1,21 +1,44 @@
+# Melissa Core Dependencies
+wheel==0.24.0
+six==1.10.0
+oauthlib==1.0.3
+requests-oauthlib==0.5.0
+Wave==0.0.2
+PyAudio==0.2.9
+pygame==1.9.2b6
```

Comment:
No, I meant move just the pygame and not the others. Also, please rebase to a single commit after making your changes.